elyra/pipeline/pipeline.py

```diff
@@ -368,11 +376,19 @@ def runtime_config(self) -> str:
     def operations(self) -> Dict[str, Operation]:
         return self._operations
 
+    @property
+    def description(self) -> str:
```

Comment:
NIT: I think this should be:
```suggestion
    def description(self) -> Optional[str]:
```

Reply:
Thank you! Fixed!